hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClientConnectionManagerImpl.java

```diff
@@ -290,16 +290,16 @@ public Connection getActiveConnection(Address target) {
 
     @Override
     public Connection getOrConnect(Address address) throws IOException {
-        return getOrConnect(address, false);
+        return getOrConnect0(address, false);
     }
 
     @Override
-    public Connection getOrTriggerConnect(Address target, boolean acquiresResources) throws IOException {
+    public Connection getOrAsyncConnect(Address target, boolean acquiresResources) throws IOException {
```

Comment:
I think that trigger is more explanatory and matches better with the intent of the method rather than the name async.

Reply:
I do not agree. You are asynchronously connecting.